Content.Server/VendingMachines/VendingMachineEmaggedSystem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Shared.Emag.Systems;
+using Content.Shared.VendingMachines.Components;
+
+namespace Content.Server.VendingMachines;
+
+public sealed class VendingMachineEmaggedSystem: EntitySystem
```

Comment:
```suggestion
public sealed class VendingMachineEmaggedSystem : EntitySystem
```

Fix this for all classes.